### PR TITLE
Enable some of the other options

### PR DESCRIPTION
**Pull request by raskchanky**
In this PR, I've enabled the options from my last PR that felt a bit risqué, along with a commit to see how they actually look in our code.

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-api-client/src/error.rs [hunk 1]
```diff
@@ -54,11 +54,11 @@ impl fmt::Display for Error {
             Error::APIError(ref c, ref m) if !m.is_empty() => format!("[{}] {}", c, m),
             Error::APIError(ref c, _) => format!("[{}]", c),
             Error::BadResponseBody(ref e) => format!("Failed to read response body, {}", e),
-            Error::DownloadWrite(ref p, ref e) => format!(
-                "Failed to write contents of builder response, {}, {}",
-                p.display(),
-                e
-            ),
```
Comment (raskchanky) [inline components/builder-api-client/src/error.rs]: This is an example of where I think `force_multiline_blocks = true` results in more readable code.

File: components/builder-api-client/src/lib.rs [hunk 2]
```diff
@@ -86,26 +86,26 @@ impl fmt::Display for NetError {
...
 #[derive(Clone, Deserialize)]
 pub struct OriginSecret {
-    pub id: String,
+    pub id:        String,
     pub origin_id: String,
-    pub name: String,
-    pub value: String,
+    pub name:      String,
+    pub value:     String,
 }
```
Comment (raskchanky) [inline components/builder-api-client/src/lib.rs]: I ❤️ this

File: components/builder-api-client/src/lib.rs [hunk 3]
```diff
@@ -218,41 +214,39 @@ pub struct OriginPrivateSigningKey {
...
-                origin: ident.origin,
-                name: ident.name,
-                version: Some(ident.version),
-                release: Some(ident.release),
-            }
+            super::PackageIdent { origin:  ident.origin,
+                                  name:    ident.name,
+                                  version: Some(ident.version),
+                                  release: Some(ident.release), }
         }
```
Comment (raskchanky) [inline components/builder-api-client/src/lib.rs]: I think this is `indent_style = "Visual"` in action, and honestly, I could go either way on this.

File: components/builder-api-client/src/lib.rs [hunk 4]
```diff
@@ -268,14 +262,12 @@ pub trait DisplayProgress: Write {
...
-    ) -> Result<Self>
-    where
-        U: IntoUrl,
+    pub fn new<U>(endpoint: U,
+                  product: &str,
+                  version: &str,
+                  fs_root_path: Option<&Path>)
+                  -> Result<Self>
+        where U: IntoUrl
     {
```
Comment (raskchanky) [inline components/builder-api-client/src/lib.rs]: I think this is also `indent_style = "Visual"` though, and in this case, I think it's a big improvement.

File: components/builder-api-client/src/lib.rs [hunk 5]
```diff
@@ -412,37 +400,31 @@ impl Client {
...
-        promote: bool,
-    ) -> Result<()> {
+    pub fn job_group_promote_or_demote<T: AsRef<str> + serde::Serialize>(&self,
+                                                                         group_id: u64,
+                                                                         idents: &[T],
+                                                                         channel: &ChannelIdent,
+                                                                         token: &str,
+                                                                         promote: bool)
+                                                                         -> Result<()> {
         let json_idents = json!(idents);
```
Comment (raskchanky) [inline components/builder-api-client/src/lib.rs]: This feels a little extreme in terms of width.

File: components/builder-api-client/src/lib.rs (hunk 3 above)
Comment (baumanj) [reply components/builder-api-client/src/lib.rs]: Yeah, both look fine to me

File: components/builder-api-client/src/lib.rs (hunk 5 above)
Comment (baumanj) [reply components/builder-api-client/src/lib.rs]: Agreed; can we tune this down a little bit to get the nice stuff but not go this far?

File: components/builder-api-client/src/lib.rs (hunk 2 above)
Comment (christophermaier) [reply components/builder-api-client/src/lib.rs]: oooooh, that's fantastic

**Pull request merged**